src/neural/onnx/converter.cc

```diff
@@ -148,6 +162,7 @@ class Converter {
   const pblczero::Net& src_;
   const WeightsToOnnxConverterOptions& options_;
   ActivationFunction default_activation_;
+  float default_eps_;
```

Comment:
I'd make `default_eps_` and `default_activation_` const (so that when they are forgotten to be initialized, it's a compile-time error), and would initialize them in the initializer list of the constructor.